runtime/src/app/router/index.ts

```diff
@@ -68,7 +67,7 @@ export function init(base: string, handler: (dest: Target) => Promise<void>): vo
 }
 
 export function extract_query(search: string) {
-	const query = Object.create(null);
+	const query: Record<string, string | string[]> = Object.create(null);
```

Comment:
can we keep the `Query` type?

Reply:
I think it should also be used in `PageContext`?

Reply:
Fair enough. I put it back together with `Params` that I renamed `PageParams`